src/qt/forms/helpmessagedialog.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>800</width>
-    <height>400</height>
+    <width>585</width>
```

Comment:
yes. Much less content is shown, especially on the 'about' dialog this resize makes sense.